Content.Server/Administration/Systems/BwoinkSystem.cs

```diff
@@ -418,7 +425,23 @@ protected override void OnBwoinkTextMessage(BwoinkTextMessage message, EntitySes
             if (_playerManager.TryGetSessionById(message.UserId, out var session))
             {
                 if (!admins.Contains(session.ConnectedClient))
-                    RaiseNetworkEvent(msg, session.ConnectedClient);
+                {
+                    // If _overrideClientName is set, we generate a new message with the override name. The admins name will still be the original name for the webhooks.
+                    if (_overrideClientName != string.Empty)
+                    {
+                        var overrideMsgText = senderAdmin switch // Theres probably a better way to do this.
+                        {
+                            var x when x is not null && x.Flags == AdminFlags.Adminhelp =>
+                                $"[color=purple]{_overrideClientName}[/color]: {escapedText}",
```

Comment:
Please just use an if statement it's more readable.

Reply:
Done